client/client.ts

```diff
@@ -164,6 +163,37 @@ function decryptResponse(t: any) {
     b.setTile(Tile.fromJSON(t));
 }
 
+/*
+ * Get signature for move proposal. This signature and the queued move will be
+ * sent to the chain for approval.
+ */
+async function proposeResponse(sig: string, uFrom: any, uTo: any) {
```

Comment:
nicee makes sense to me 

the structure here will likely change during the integration to handle multiple pending moves. MUD clients have this optimistic rendering feature, where you can have multiple moves submitted at once, and they all are handled in the background while the user display assumes they all just work 

rn this blocks on a single move, which is perfect for now

Reply:
sg! yeah a lot of what I did here is temporary. aside from setting up the alchemy node, do you think maybe we should hold off on thinking about rendering until we see small brain's front-end?

Reply:
completely agree with you. holding off on rendering is the right move